subprojects/js/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 mkdir -p build
-#(cd build && emcmake cmake -DCMAKE_BUILD_TYPE=Debug ..)
-(cd build && emcmake cmake ..)
```

Comment:
This makes the default build is debug.
Is that the intention? can you elaborate?